Backend/Lyzer/Services/ResultsService.cs

```diff
@@ -0,0 +1,42 @@
+using System.Runtime.Serialization;
+using Lyzer.Clients;
+using Lyzer.Common.Constants;
+using Lyzer.Common.DTO;
+using Newtonsoft.Json;
+
+namespace Lyzer.Services
+{
+    public class ResultsService
+    {
+        private readonly ILogger<ResultsService> _logger;
+        private readonly JolpicaClient _client;
+        private readonly CacheService _cache;
+
+        public ResultsService(ILogger<ResultsService> logger, JolpicaClient client, CacheService cache) 
+        {
+            _logger = logger;
+            _client = client;
+            _cache = cache;
+        }
+
+        public async Task<ResultsDTO> GetLatestRaceResult() {
```

Comment:
Let's also maybe change this to be GetRaceResult with a string year as a parameter. Reduces a need to update it later on when we are wanting to get the race results for a particular year. (Same comment as the one that is in jolpica client.
This can then be called in wherever, passing through the year

Reply:
Same here as with the Client comment.

Reply:
Following up to the conversation on the controller,
I could change change it to get `GetCachedRaceResult` with a parameter in case we ever need to cache results for other years/rounds.

But so far, I am concerned about creating a function called `GetRaceResult` that in future we will build upon with database stuff upon a function that already feels like its doing its fair bit with caching.